src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/mapping/BurdenMappingHelper.kt

```diff
@@ -37,4 +39,25 @@ class BurdenMappingHelper : MappingHelper()
                 problems
         )
     }
+
+    fun mapModelParameterValuesPlain(record: Record): ModelRunParametersValue
+    {
```

Comment:
I'd rename this model from`ModelRunParametersValue` to just `ModelRunParameter`, and this method to `mapModelRunParameter`